doc/adr/0006-use-git-commits-to-explain-our-work.md

```diff
@@ -0,0 +1,103 @@
+# 6. Use Git commits to explain our work
+
+Date: 2022-10-12 (retroactive)
```

Comment:
I wanted it to convey that we'd been working this way since the start of the project, but I also got the year wrong somehow so this was a right old mess. Have just made it today's date.